source/gui/widgets/listbox.cpp

```diff
@@ -5319,6 +5324,14 @@ namespace nana
 			return pos;
 		}
 
+		void listbox::clear_headers()
+		{
+			internal_scope_guard lock;
+			auto & ess = _m_ess();
+			ess.header.clear();
```

Comment:
I think it is better that all items are deleted as well

Reply:
Yes I think it's better too :+1:

Reply:
So, would you like to update the PR? Im looking forward to it.

Reply:
Added ess.lister.erase(); to erase all the items.